lib/open-search-client.js

```diff
@@ -195,7 +195,7 @@ const createTermsIndex = (prefixUrl, index) => createIndex(prefixUrl, index, {
       source_terminology: {
         type: 'keyword',
       },
-      graphUri: {
+      ontologyGraph: {
```

Comment:
Looks like this is probably the merge conflict. Any particular reason for this change?

Reply:
Updated in 33fae9e